tools/npm/npm1.md

```diff
@@ -0,0 +1,158 @@
+# Definition
```

Comment:
let's find a better name :) like 'what-is-npm.md' or just 'npm' or 'npm-guide' or .... You get the point ;)